RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -361,9 +359,9 @@ private void ExecuteQuickFixInModuleCommand(object parameter)
                 .ToList();
 
             var items = filteredResults
-                .Where(result => !(result is AggregateInspectionResult))
+                .Where(result => result.GetType().Name != "AggregateInspectionResult")
```

Comment:
makes stuff funny when renaming the type ... 
We can't get the dependency in here and making this a special case smells, because we're violating LSP by special-casing this `IInspectionResult`...

This logic should be encapsulated into Inspection results in the first place:

     var items = filteredResults
         .Select(item => item.DefaultQuickFix)
         .OrderByDescending(fix = fix.Selection);

also a good moment to rename items to something like `defaultQuickFixes`